src/js/jwt/jwt.js

```diff
@@ -288,21 +288,29 @@ function refreshTokens() {
 
 // Actually update the token
 function updateToken() {
-  return priv.keycloak.updateToken().then((refreshed) => {
-    // Important! after we update the token
-    // we have to again populate the Cookie!
-    // Otherwise we just update and dont send
-    // the updated token down stream... and things 401
-    setCookie(priv.keycloak.token);
-
-    log('Attempting to update token');
-
-    if (refreshed) {
-      log('Token was successfully refreshed');
-    } else {
-      log('Token is still valid, not updating');
-    }
-  });
+  return priv.keycloak
+    .updateToken()
+    .then((refreshed) => {
+      // Important! after we update the token
+      // we have to again populate the Cookie!
+      // Otherwise we just update and dont send
+      // the updated token down stream... and things 401
+      setCookie(priv.keycloak.token);
+
+      log('Attempting to update token');
+
+      if (refreshed) {
+        log('Token was successfully refreshed');
+      } else {
+        log('Token is still valid, not updating');
+      }
+    })
+    .catch((err) => {
+      log(err);
+      Sentry.captureException(new Error('Update token failed'));
```

Comment:
maybe it'd be useful to log the actual exception in Sentry so we'll have more context if it ever happens?